gramps/plugins/webreport/addressbook.py

```diff
@@ -84,12 +84,13 @@ def __init__(self, report, title, person_handle, has_add, has_res, has_url):
 
         # set the file name and open file
         output_file, sio = self.report.create_file(person_handle, "addr")
-        addressbookpage, head, body = self.write_header(_("Address Book"))
+        result = self.write_header(_("Address Book"))
+        addressbookpage, head, body, outerwrapper = result
```

Comment:
You can indicate that a variable is expectedly unused by adding an underscore.  See the regexp in the pylint [configuration](https://github.com/gramps-project/gramps/blob/master/.pylintrc#L290) file.

Consider using `body_` instead of `body`.

Reply:
I don't think that works. I replaced body by body_
worst, I have new errors.